test/test_mutation.jl

```diff
@@ -1,4 +1,30 @@
 using Test
+using GeneticAlgorithms.Mutation
+
+
+@testset "Mutation.jl" begin
+
+    mutation = RealGeneMutation(0.1, (-0.5, 0.5))
+    mutated_genes = mutation([0.0 0.0 0.0 0.0 0.0])
+    @test length(mutated_genes) == 5
+    for gene in mutated_genes
+        @test -0.5 <= gene <= 0.5
+    end
+    @test mutation.mutation_rate == 0.1
+    @test mutation.mutation_interval == (-0.5, 0.5)
+
+    mutation_int = RealGeneMutation(0.9, (0,17))
+    mutated_genes = mutation_int([0 0])
+    @test all(mutated_genes .>= 0 .& mutated_genes .<= 17)
+
```

Comment:
Amybe orient your tests on `rouletteWheelSelection` where a rand_generator is passed to the function. This allows you wirte more meaningful tests